httphandler/main.go

```diff
@@ -1,11 +1,28 @@
 package main
 
 import (
+	"context"
+	"net/url"
+	"os"
+
 	logger "github.com/kubescape/go-logger"
+	"github.com/kubescape/kubescape/v2/core/cautils"
 	_ "github.com/kubescape/kubescape/v2/httphandler/docs"
 	"github.com/kubescape/kubescape/v2/httphandler/listener"
 )
 
 func main() {
-	logger.L().Fatal(listener.SetupHTTPListener().Error())
+	ctx := context.Background()
+	// to enable otel, set OTEL_COLLECTOR_SVC=otel-collector:4317
+	if otelHost, present := os.LookupEnv("OTEL_COLLECTOR_SVC"); present {
+		ctx = logger.InitOtel("kubescape",
+			os.Getenv(cautils.BuildNumber),
+			os.Getenv("ACCOUNT_ID"),
```

Comment:
[Required]: Load ID from a local file (mounted via configMap)

Reply:
checking at `customerloader.go` I see there might be an env `KS_ACCOUNT_ID` for that too?

Reply:
Yes, KS supports `KS_ACCOUNT_ID` env. The account ID can be found at `/home/ks/.kubescape/config.json`. Please take a look at how Kubescape loads the accountID.